Aspose.Slides.WebExtensions/Aspose.Slides.WebExtensions.csproj

```diff
@@ -132,7 +132,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Aspose.Slides.NET">
-      <Version>21.11.0</Version>
+      <Version>23.12.0</Version>

```

Comment:
Hi @polonair ,

Please also update version in examples projects.

Reply:
Done